qa/util/src/main/java/io/camunda/qa/util/cluster/TestRestOperateClient.java

```diff
@@ -153,6 +153,36 @@ public Either<Exception, HttpResponse<String>> resolveIncident(final long proces
         processInstanceKey, new CreateOperationRequestDto(OperationType.RESOLVE_INCIDENT));
   }
 
+  public HttpResponse<String> getRequest(final String endpointUriFormat, final String key)
+      throws URISyntaxException, IOException, InterruptedException {
+    final String url = endpoint + endpointUriFormat.formatted(key);
+    final HttpRequest request = addAuthHeader(createBuilder(url)).GET().build();
+    return httpClient.send(request, BodyHandlers.ofString());
+  }
+
+  public HttpResponse<String> getRequest(final String endpointUriFormat, final long key)
+      throws URISyntaxException, IOException, InterruptedException {
+    return getRequest(endpointUriFormat, String.valueOf(key));
+  }
```

Comment:
👍 This is a nicely generalized method.

🤔 I think we can refactor this class a bit further, and re-use this method in the other HTTP GET endpoints. In the future, if we need to introduce changes to all of the HTTP GET endpoints, we only need to modify a single, "root" method. Otherwise, every time we refactor this class, we'll need to waste time to understand how each operation method formats it's URL and sends the request.

🔧 Alternatively, if we can't re-use these new methods, maybe we can give them more specific names (like `getEntityRequest` maybe?), so that it's clear it's not a helper method for all HTTP GET requests.